VideoApi/VideoApi.DAL/Mappings/ParticipantMap.cs

```diff
@@ -8,13 +8,15 @@ public class ParticipantMap : IEntityTypeConfiguration<Participant>
     {
         public void Configure(EntityTypeBuilder<Participant> builder)
         {
-            builder.Property(x => x.FirstName);
-            builder.Property(x => x.LastName);
             builder.Property(x => x.ContactEmail);
-            builder.Property(x => x.ContactTelephone);
-            builder.Property(x => x.CaseTypeGroup);
-            builder.Property(x => x.Representee);
             builder.Property(x => x.CurrentRoom);
+            
+            // Keep deleted properties as columns in the table
```

Comment:
would it be safer to set the column as nullable or default value of null?

Reply:
they are already nullable. Have checked via the automation tests - no issues creating a participant without those values populated